apps/full-site-editing/package.json

```diff
@@ -16,10 +16,10 @@
 	},
 	"homepage": "https://github.com/Automattic/wp-calypso",
 	"scripts": {
-		"plugin": "webpack --source='plugin'",
+		"plugin": "cb --config='./webpack.config.js' --source='plugin' --env.WP",
```

Comment:
Do you know where these `--source` arguments come from? I don't see them documented in [webpack CLI](https://webpack.js.org/api/cli/).

Reply:
Yeah, they're a custom addition to `argv` in `full-site-editing`'s Webpack config:

https://github.com/Automattic/wp-calypso/blob/acfc8a24bb929b72ef109e7d1d0e3ef7ed6eaafb/apps/full-site-editing/webpack.config.js#L28

Was also wondering if we should discourage :thinking:

Reply:
> Was also wondering if we should discourage 🤔

Oh, that's interesting! I'd say `cb` itself should be very strict about adhering to the webpack CLI, but let's not prescribe how consumers should use it (yet)…

Reply:
Some feedback about naming:
- `cb` feels like something very generic, like a shell command. Specialized commands should have longer names, e.g., `calypso-build`.
- `webpack --source`: again, very generic, looks like a built-in webpack option. `--calypso-something-source` would communicate better that it's something custom.

Reply:
Thanks Jarda! I tend to agree with both points.

For a while, I was thinking that `cb` was a nice shorthand, and that we could use `npx --package=@automattic/calypso-build cb` to make sure we're getting the right command, but using that caused some rather annoying that I wasn't able to resolve easily.

Inclined to file a commit to `s/cb/calypso/build/g`, unless anyone objects? @sirreal?

Reply:
> * `cb` feels like something very generic, like a shell command. Specialized commands should have longer names, e.g., `calypso-build`.

@blowery suggested this in the interest of brevity. In practice, this will likely live mostly inside `package.json` `scripts` 🤔 I'm fine either way.

> * `webpack --source`: again, very generic, looks like a built-in webpack option. `--calypso-something-source` would communicate better that it's something custom.

This isn't a "core" `calypso-build` option, but some configuration that a particular webpack config function is exposing which is why I'm not too concerned. It would be harmful if folks start copying the pattern and expect it to be "core" functionality… Perhaps it is something to discourage?

Reply:
> * `webpack --source`: again, very generic, looks like a built-in webpack option. `--calypso-something-source` would communicate better that it's something custom.

Agree. We're using `env` (e.g. `env.WP`) to communicate Calypso-specific stuff, whereas `full-size-editing` deviates from that pattern. However, the `--source` arg has been merged prior to this PR and is thus arguably orthogonal to this one, so I'd like to change it with a separate PR.